src/styles/components/_c-map.scss

```diff
@@ -1,3 +1,9 @@
 .c-map {
   height: calc(100vh - #{$nav-height} - #{$hero-height} - #{$footer-height});
-}
+  &.-layers-container {
```

Comment:
Not sure about this. Should be its own independant component placed separately. Like zoom controls etc.